antsibull/dependency_files.py

```diff
@@ -155,7 +155,19 @@ def write(self, ansible_version: 'PypiVer', ansible_base_version: str,
         """
         records = []
         for dep, version in dependencies.items():
-            records.append(f'{dep}: >={version.major}.{version.minor}.0,<{version.next_major()}')
+            if version.prerelease and version.minor == 0 and version.patch == 0:
```

Comment:
I think this should work:
```suggestion
            if version.prerelease and version.patch == 0:
```